Description: Skip source file in patch directory when checking
 internationalization.  This file was causing the dh_auto_test step to fail.
 .
 This patch is only necessary because the package uses quilt, so it is not
 appropriate for upstream. -- Richard Laager <rlaager@wiktel.com>
Author: tony mancill <tmancill@debian.org>
Forwarded: not-needed
Last-Update: 2012-03-18

--- /dev/null
+++ gbonds-2.0.3/po/POTFILES.skip
@@ -0,0 +1 @@
+.pc/libgnomeprint/src/commands.c
